src/core/adapters/adaptdirectanswer.js

```diff
@@ -0,0 +1,33 @@
+import DirectAnswer from '../models/directanswer';
+/**
+ * Constructs an SDK DirectAnswer from an answers-core DirectAnswer
+ *
+ * @param {DirectAnswer} directAnswer from answers-core
+ * @returns {@link DirectAnswer}
+ */
+export default function adaptDirectAnswer (directAnswer) {
+  if (!directAnswer) {
+    return new DirectAnswer();
+  }
+
+  const relatedResult = directAnswer.relatedResult || {};
+
+  return new DirectAnswer({
```

Comment:
would it be possible to update direct answer to just use the new core's model? I'm pretty sure direct answer is universal results only. Also down for making it a separate pr!

Reply:
ya, that actually may be simpler in this case. I can handle it in this PR